static/js/project/controllers/project.controller.js

```diff
@@ -47,10 +49,41 @@
                     $mdToast.showSimple('Could not get project.');
                 }
             ).finally(function () {
+                getAWS();
+            });
+        }
+
+        function getAWS() {
+            User.get_aws_account().then(
+                function success(response) {
+                    self.aws_account = response[0];
+                },
+                function error(response) {
+
+                }
+            ).finally(function () {
+
+            });
+        }
+
+        function create_or_update_aws() {
+            if (self.aws_account.client_secret == null || self.aws_account.client_id == null) {
+                $mdToast.showSimple('Client key and secret are required');
+            }
+            User.create_or_update_aws(self.aws_account).then(
```

Comment:
This should run when values are provided. 

Is there a way we can validate if client key and secret are valid?  
If no, we should handle wrong key and secret.